packages/analytics/amplify_analytics_pinpoint_dart/test/queued_item_store_test.dart

```diff
@@ -0,0 +1,243 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import 'package:amplify_analytics_pinpoint_dart/src/impl/analytics_client/event_client/queued_item_store/dart_queued_item_store.dart';
+
+import 'package:test/test.dart';
+
+void main() {
+  late DartQueuedItemStore db;
+
+  group('DartQueuedItemStore Tests', () {
+    setUpAll(() {
+      db = DartQueuedItemStore('test');
+    });
+
+    /// Reload db instance
+    Future<void> refreshDb() async {
+      await db.clear();
+    }
+
+    setUp(() async {
+      await refreshDb();
```

Comment:
nit: can just call `await db.clear()` as `refreshDb` is not used anywhere else :D